okdata/cli/commands/permissions.py

```diff
@@ -11,9 +11,9 @@ class PermissionsCommand(BaseCommand):
     __doc__ = f"""Oslo :: Permissions
 
 Usage:
-  okdata permissions ls [<resource_name>] [--format=<format> --env=<env>]
-  okdata permissions add <resource_name> <user> <scope> [--team | --client] [--env=<env>]
-  okdata permissions rm <resource_name> <user> [<scope>] [--team | --client] [--env=<env>]
```

Comment:
`--env=<env>` må erstattes med `options` i disse for å få `--env` til å fortsette å fungere med de kommandoene.

Reply:
Ah det gikk jeg glipp av 🗡️ skal fikse